File: problems/intermediate/power_ranger/_tests_/power_ranger.js
const powerRanger = require("../power_ranger")

function test() {
    if (powerRanger.getPowerRange(3, 1, 27)!=3) {
        throw console.error("❌ failed js test 1");
        
    }
    if (powerRanger.getPowerRange(2, 1, 100)!=10) {
        throw console.error("❌ failed js test 2");
    }
    else{
        console.log("✅ passed all js tests congrats");
    }
}

test();

File: problems/intermediate/power_ranger/_tests_/power_ranger.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from power_ranger import getPowerRange

def test():
    if getPowerRange(2, 49, 65) != 2:
        print("❌ failed python test 1")
    if getPowerRange(4, 250, 1300) != 3:
        print("❌ failed python test 2")
    else:
        print("✅ passed all python tests congrats")


if __name__ == "__main__":
    test()